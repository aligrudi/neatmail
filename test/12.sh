# neatmail test
./mail pg -h from: -h subject: $1 1
