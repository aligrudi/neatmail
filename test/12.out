From: B <b@b.eu>
Subject: Message from B

This is B.

